src/App.js

```diff
@@ -13,7 +14,9 @@ function App() {
       </header>
       <main>
         <PasswordInput password={password} updatePassword={updatePassword} />
-        <PasswordIndicatorGrid password={password} />
+        <Suspense fallback={<PasswordIndicatorGridPlaceholder />}>
+          <PasswordIndicatorGrid password={password} />
+        </Suspense>
```

Comment:
this is not strictly necessary but it's a new-ish react feature I wanted to try out. [Docs link](https://reactjs.org/docs/code-splitting.html). Basically, we split out the password indicator grid into its own junk of JS and then load it asynchronously from the rest of the page; that way our main bundle stays small and loads fast, and we can use fat third party dependencies (like zxcvbn) without sinking our page load time.

To try out:
* run `npm run build` to build a production bundle (this doesn't work when using the webpack dev server)
* run `npx -s serve build` to serve the production bundle at http://localhost:5000
* Navigate to that page, and, optionally, enable throttling for the connection in the "network" tab of the browser developer tools so see what it looks like on a slow connection